src/phoenix/trace/exporter.py

```diff
@@ -69,3 +89,12 @@ def _send(self, span: Span) -> None:
             self._session.post(self._url, data=data)
         except Exception as e:
             logger.exception(e)
+
+    def _raise_if_phoenix_is_not_running(self) -> None:
+        try:
+            requests.get(f"{self._base_url}/arize_phoenix_version").raise_for_status()
+        except Exception:
+            raise Exception(
+                f"Arize Phoenix is not running on {self._base_url}. Launch Phoenix "
+                f"with `import phoenix as px; px.launch_app()`"
+            )
```

Comment:
An exception seems too strong. Can we just log a warning?

Reply:
ok. will change